hass-addon-sunsynk-edge/config.yaml

```diff
@@ -59,7 +59,7 @@ schema:
       MODBUS_ID: int(1,16)
       DONGLE_SERIAL_NUMBER: str?
       PORT: str
-  SENSOR_DEFINITIONS: list(single-phase|three-phase|three-phase-hv)
+  SENSOR_DEFINITIONS: list(single-phase|single-phase-16Kw|three-phase|three-phase-hv)
```

Comment:
Either all names needs to be lowercase, or we need a `.lower()` in the load function in definitions/__init__.py

Kw will not match the filename kw today

Reply:
Codecov should be taken care of by a previous commit. It imports and tests all the existing definition files.